.travis.yml

```diff
@@ -4,18 +4,24 @@ branches:
   - develop
   - /^issue.*$/
 
-notifications:
-  slack:
-    secure: "BsuSdYLxqxthz02FINOsPXzR7Caxt7PrvEszcGN3PsWanXPmR5yM3zH9k4rX3gvGV3XJAS5SZx2HuTiI82n8N/jIyE32cJpycyZUgMzhviVzCPzjthfoG2nTf3oAbd2k1tMHCmT92cJ13dbwm4lnVKugVFQL3sEHUr1UknD8/G5EKmqC/wdYO/QBQDv8QvXCi4XPbEVOkfVI6Toh/wgW/PQk98VSMYentYL27gDYBf3TwN1lWxFW37EKgG9wyjCV4kZsGLLhDH+ppo5Y6RWG1zqB8b/qz/wXK1PoDbfFlvqm1M333qS9DtZIjRGAQNkfelrg2KhZnNEDOAPr9gpqx/HGuz9jPn2SRN98sKouQf8E+IR+5pUXC5Q8nCouBeGe6XZB1Af7NWkp5VM80YOR5Ss97HXH7Yw6KXFhdy+i5mg241Hxxl0zCS8nqvNBdzMAWmqLjm481YvQGuFrGvwyl9mQYJ5xYx90TiOixWzsB9TGEb9uyirPXVozEYtGXuqtD32VpV1UA3q9A+JLYTDadKT/Sd6qEnr4+eSL96PW6ZFjFF/+HKauuJ30GfNPANj1V2gd0RlY542gLV44/9a7kT/yyxMKvrR6cnLjTyZCVIoxB1q342wQP88Re/Meehp3QsV1/qHOBVe4VxAiQSIKjjLFZ9ahMZN8yLwTpA+McyU="
-
 matrix:
   include:
     - os: linux
       dist: trusty
       sudo: required
+      env: SWIFT_SNAPSHOT=4.1
```

Comment:
To be consistent with our other Travis files, we should remove the `SWIFT_SNAPSHOT=4.1` which means we'll instead use the `.swift-version` (which is now 4.1).